lib/adal.js

```diff
@@ -799,7 +796,13 @@ var AuthenticationContext = (function () {
     AuthenticationContext.prototype.promptUser = function (urlNavigate) {
         if (urlNavigate) {
             this.infoPii('Navigate to:' + urlNavigate);
-            window.location.replace(urlNavigate);
+            if (this.config.displayCall) {
```

Comment:
`displayCall` is moved here so we can use it for all navigation. This is useful when we are running in an environment where we don't have access to `window`